configs/mnedc/client-config.yaml

```diff
@@ -0,0 +1,3 @@
+# This configuration file is for providing the Actual IP and Port of the MNEDC server to the clients
```

Comment:
Please add a note that `server-ip` and `port` need to be modified for each environment.